testsuite/run_sets/github_validation/github_validation_proxy.yml

```diff
@@ -8,11 +8,8 @@
 
 # IMMUTABLE ORDER
 
-# Proxy
-  # one of: proxy_register_as_minion_with_script.feature
-  #         proxy_register_as_minion_with_gui.feature
-  #         proxy_register_as_pod.feature
-- features/init_clients/proxy_register_as_minion_with_script.feature
+- features/init_clients/proxy_traditional.feature
+- features/init_clients/proxy_container.feature
```

Comment:
We need to also adjust the GitHub PR tests for this, otherwise they will break I guess.

Reply:
No, is fine, what I do is to skip the whole feature depending the environment.